releases/release-1.24/release-notes/release-notes-draft.json

```diff
@@ -565,6 +706,39 @@
     ],
     "duplicate": true
   },
+  "106539": {
+    "commit": "6de9dddf94c75a19d8baadf9e22f94095956ee2b",
+    "text": "Replace the url label of rest_client_request_duration_seconds and rest_client_rate_limiter_duration_seconds metrics with a host label to prevent cardinality explosions and keep only the useful information. This is a breaking change required for security reasons.",
```

Comment:
```suggestion
    "text": "Replace the url label of `rest_client_request_duration_seconds` and `rest_client_rate_limiter_duration_seconds` metrics with a host label to prevent cardinality explosions and keep only the useful information. This is a breaking change required for security reasons.",
```